Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorHollowEffect.py

```diff
@@ -107,17 +95,9 @@ def updateGUIFromMRML(self):
     selectedSegmentLabelmapSpacing = [1.0, 1.0, 1.0]
     selectedSegmentLabelmap = self.scriptedEffect.selectedSegmentLabelmap()
     if selectedSegmentLabelmap:
-      selectedSegmentLabelmapSpacing = selectedSegmentLabelmap.GetSpacing()
-      kernelSizePixel = self.getKernelSizePixel()
-      if kernelSizePixel[0]<=1 and kernelSizePixel[1]<=1 and kernelSizePixel[2]<=1:
-        self.kernelSizePixel.text = "requested. Not feasible at current resolution."
-        self.applyButton.setEnabled(False)
-      else:
-        thicknessMm = [abs((kernelSizePixel[i]-1.0)/2.0*selectedSegmentLabelmapSpacing[i]) for i in range(3)]
-        self.kernelSizePixel.text = "requested. Actual: {0} x {1} x {2} mm.".format(*thicknessMm)
-        self.applyButton.setEnabled(True)
+      self.marginInfoLabel.text = ""
```

Comment:
Actual shell size is still constrained by the voxel size, so it would be important to keep displaying the request and actual voxel size separately.

Reply:
Can you elaborate how the method used for stepSize can be applied here?

Reply:
We could round the displayed "actual margin" value to a few decimals, as it is done for stepSize value.

3 floating-point numbers can be quite long, even if rounded, so maybe we could display it in a separate row.